_sass/_custom_classes.scss

```diff
@@ -87,6 +87,10 @@ footer {
     background: $red;
 }
 
+.bg-primary-light {
+    background: $primary-light;
+}
+
```

Comment:
This is defining a CSS class for a background that uses the new `primary-light` color.